.circleci/docker/build.sh

```diff
@@ -274,19 +274,19 @@ case "$image" in
     VISION=yes
     KATEX=yes
     ;;
-  pytorch-linux-bionic-rocm3.9-py3.6)
+  pytorch-linux-bionic-rocm3.10-py3.6)
```

Comment:
Since 3.9 docker image is still being build (see line 6623 of config.yaml), you can just replace 3.9 with 3.10 here.

Reply:
Fixed.